input/Hi.in

```diff
@@ -56,6 +56,17 @@
      }
    }
 
+   Group {
+     list = [ "conserved", "make_field_conservative" ];
+     conserved {
+         field_list = [ "density", "velocity_x", "velocity_y",
+                        "internal_energy", "total_energy"  ];
+     }
+     make_field_conservative {
+         field_list = [ "velocity_x", "velocity_y",
+                        "internal_energy", "total_energy" ];
+     }
```

Comment:
From a physical point of view, the total mass, momenta and total energy should be conserved.
Am I misinterpreting the naming (and could be make this potentially more clear) or should different fields be conserved?
Potentially related to #56

Reply:
You're right, these fields are those whose fluxes are provided by hydro and are required in maintaining conservation, but not necessarily conserved themselves.  And, given that specifying field names in parameter files is being depreciated, these field groups should not be required at all.  I'll deal with the first issue for this PR, and defer the second.